generators/spa/templates/VueJS/common/Settings.html

```diff
@@ -1,19 +1,27 @@
 [JavaScript:{ jsname: "JQuery" }]
-[JavaScript:{ path: "https://cdn.jsdelivr.net/npm/vue/dist/vue.js"}]
+[JavaScript:{ jsname: "Knockout" }]
```

Comment:
@VAlmea What are we using `knockout` for here?  Should we be using `knockout` at all?

Reply:
@VAlmea Please respond.

Reply:
The main page is the only SPA. The settings view is the same one used for all other modules. The reference to Knockout was already implemented

Reply:
@WillStrohl I Tested the configuration view and it is not necessary to reference the Knockout library. I have removed it